### DISCUSSION TITLE
Osu Lazer will not update for me

### DISCUSSION THREAD

**Discussion by MordokSonny** [ppy/osu#30821] [Q&A] [answered]
The version of my game is 2024.1115.1, I am on a samsung galaxy S21+. I have downloaded the update 2024.1115.3(if that is correct) multiple times now and every time I do, the version miraculously shows 2024.1115.1. I know im not downloading that version. I had accidently deleted the game off my phone and lost every song I had a few days ago and even that didn't fix the problem. I will provide a video of me downloading version 2024.1115.3 and the screenshot is showing the version it says it is after I download it.

https://github.com/user-attachments/assets/82bfd7e4-e9f1-40dc-80af-657eb1172f1e

![Screenshot_20241121_162806_ZArchiver](https://github.com/user-attachments/assets/1e3171d7-5bff-4019-a0a8-b50a00d6d266)

**Comment by peppy** [comment, marked as answer]
It's fine, you're on the latest. Please wait for the next update and this prompt will stop appearing.

**Reply by Thepupman1** [reply]
Thank you! Do you know why version 2024.1115.3 gets downloaded as 2024.1115.1, though? It is because it's meant for Linux, so android can't update it to match versions (since 2024.1115.3 doesn't exist for android)?

**Reply by peppy** [reply]
We uploaded it as such. We're working on fixing android releases and can't build a new one right now.
